modules/javafx.fxml/src/test/java/test/javafx/fxml/FXMLLoader_ScriptTest.java

```diff
@@ -39,7 +38,7 @@
     @SuppressWarnings("deprecation")
     public void testStaticScriptLoad() throws IOException {
         FXMLLoader fxmlLoader = new FXMLLoader(getClass().getResource("static_script_load.fxml"));
-        FXMLLoaderHelper.setStaticLoad(fxmlLoader, true);
+        fxmlLoader.setStaticLoad(true);
```

Comment:
Since this is only called 2 times directly after the constructor it would make sense to add this directly as a param to the constructor. By doing so the field could be defined as final

Reply:
Sorry for the long gap :)

Are you trying to suggest to we can overload the constructor? Something similar to:

```
public FXMLLoader(boolean  staticLoad) {
    this.staticLoad = staticLoad;
}
```
Since FXMLoader already has a long list of constructors, do you think this would be a good idea?

Reply:
I do not recommend adding any more constructor parameters.

If you want to take this new feature forward, it  will take a fair bit of work to specify what the effect of staticLoad should be on loading fxml files. It seems like a somewhat of a SceneBuilder-specific mode, but if it has more general use then it could be something to consider. Also, there will need to be several new tests to validate the behavior of various loading scenarios when using static load, including checking what the effects are of switching between modes for successive loads. It might seem simple to "just make this method public", but there is more to it than that.

I would also want @johanvos to weigh in, since Gluon now maintains SceneBuilder.

Reply:
Thank you for the pointers. 

I will try to find if this feature has use-cases outside of Scene Builder.